transform/transformers/ImageTransformer.py

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/env python
+#   coding: UTF-8
+
+import argparse
+import datetime
+import dateutil.parser
+import glob
+from io import BytesIO
+import itertools
+import json
+import logging
+import os.path
+import shutil
+import subprocess
+import sys
+import zipfile
+
+from requests.packages.urllib3.exceptions import MaxRetryError
+
+try:
+    from PDFTransformer import PDFTransformer
+except ImportError:
+    from .PDFTransformer import PDFTransformer
+
+try:
+    from transform import settings
+    from transform.settings import session
+    from transform.views.image_filters import get_env, format_date
+except ImportError:
+    print("Available for command line operation only", file=sys.stderr)
+
+__doc__ = """
+SDX Image Transformer.
+
+Example:
+
+python transform/transformers/ImageTransformer.py --survey transform/surveys/023.0102.json \\
```

Comment:
Shouldnt use 023.0102 as the example as that is not a survey this transformer is designed to run.